<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flights</class>
 <widget class="QWidget" name="Flights">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авиарейсы</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="4">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton_add">
     <property name="text">
      <string>Добавить авиарейс</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Авиакомпания</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pushButton_delete">
     <property name="text">
      <string>Удалить рейс</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QCheckBox" name="checkBox_search">
     <property name="text">
      <string>Отфильтровать данные</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButton_change">
     <property name="text">
      <string>Изменить рейс</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Номер авиарейса</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Аэропорт прибытия</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_number"/>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_airline"/>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_arrival"/>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_departure"/>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QDateTimeEdit" name="dateTimeEdit"/>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_total_places"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Дата и время отправления</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Аэропорт отправления</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Количество мест всего</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
